pkg/crd/crd.go

```diff
@@ -639,6 +644,24 @@ func (c *Client) ListPodIds(podns, podname string) (map[string][]aadpodid.AzureI
 	return idStateMap, nil
 }
 
+// LookupClientIDByResourceID returns the clientID associated with the
+// specified resourceID defined on AzureIdentifies defined in the given
+// namespace.
+func (c *Client) LookupClientIDByResourceID(ns, resourceID string) (clientID string, err error) {
```

Comment:
We don't need this new method, because [ListPodIds](https://github.com/Azure/aad-pod-identity/blob/master/pkg/crd/crd.go#L625-L640) already gives us the `AzureIdentities` that match for the pod.